### DISCUSSION TITLE
klipper kobra neo

### DISCUSSION THREAD

**Discussion by fireman6162** [1coderookie/Klipper4KobraGoNeo#19] [Q&A]
trying to step up klipper on my neo. everytihng connects, i have control over the print  via klipper, only issue is when i try to print the printer heats up but nothing happens i have the g code in printer.cfg also pursaslicer g-code Start_Print entered in the cutsom g-code box. im stumped. ive tried most things i can think of.....home works the macros dont seem to

**Comment by 1coderookie** [comment]
Hmm, difficult to say since I don't know your specific files and setup.  
Do you get any error messages or whatsoever? Maybe you'll find something in the klippy.log, so check that as well.  
Might be easier for me to help if you'd add the printer.cfg here, paste your g-code and also upload some pictures of the g-code sections in PrusaSlicer.  
Or maybe make an account at Klipper's discourse and ask there, I guess that would be the easiest for the both of us: https://klipper.discourse.group/
